client/src/pages/Signup/Signup.js

```diff
@@ -7,49 +7,78 @@ import {
   Box,
   Button
 } from "@material-ui/core";
+import { makeStyles, createStyles } from "@material-ui/core/styles";
+import useMediaQuery from '@material-ui/core/useMediaQuery';
 
 import SignupForm from "./SignupForm";
 import SideBanner from "../../components/SideBanner";
 import { register } from "../../store/utils/thunkCreators";
 
+const useStyles = makeStyles((theme) => createStyles({
+  root: {
+    display: "flex",
+    flex: 1,
+    flexDirection: "column",
+    alignItems: "flex-start",
+    justifyContent: "flex-start",
+    height: "100%"
+  },
+  formContainer: {
+    display: "flex",
+    flex: 1,
+    width: "60%",
+    margin: "0 auto",
+    marginTop: "10rem",
+    [theme.breakpoints.down('sm')]: {
+      marginTop: "2rem",
+    },
+  }
+}));
+
 const Signup = (props) => {
```

Comment:
This component seems like it could be combined with Login as they're very similar